src/wire/ipv6hop.rs

```diff
@@ -0,0 +1,316 @@
+use core::fmt;
+use {Error, Result};
+
+pub use super::IpProtocol as Protocol;
+
+/// A read/write wrapper around an IPv6 Hop-by-Hop Options Header.
+#[derive(Debug, PartialEq)]
+pub struct HopByHop<T: AsRef<[u8]>> {
```

Comment:
`Packet` just to be in line with other implementations in `wire`.

Reply:
fixed